### PR TITLE
eslint: Fix syntax for disabling the underscore dangle rule

### PR DESCRIPTION
**Pull request by kklin**

### CODE REVIEW

File: stitch/bindings.js [hunk 1]
```diff
@@ -1,4 +1,3 @@
-/* eslint require-jsdoc: [1] valid-jsdoc: [1] */
```
Comment (kayousterhout) [inline stitch/bindings.js]: The AirBNB jslinter doesn't check either of these things (at least not in my version -- did you update it?) so the effect of removing this line is to neither warn nor error on JS doc issues.  Was this the intention / am I missing something here?  I think the right change is to change these to errors and add to the eslint config (although last I checked that would require adding some documentation to other JS files that we have)

File: stitch/bindings.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /* eslint require-jsdoc: [1] valid-jsdoc: [1] */
-/* eslint-disable no-underscore-dangle */
```
Comment (kayousterhout) [inline stitch/bindings.js]: Can you update the commit to state what the eslint version it's referring to is?

File: stitch/bindings.js (hunk 1 above)
Comment (kklin) [reply stitch/bindings.js]: Ah, I just misinterpreted. Thanks!

**Comment by kklin** [comment]
@kayousterhout updated

**Pull request merged**